maybe/syscall_filters.py

```diff
@@ -108,6 +109,30 @@ def substitute_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
+    path = abspath(path)
+    if path in allowed_files:
```

Comment:
Instead of this, it might make more sense to simply check whether the path exists and return `None` if it does, as `mknod` will fail anyway in this case and we need not intercept it.